src/style.css

```diff
@@ -0,0 +1,116 @@
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+  margin-top: 40px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  color: #616070;
+  margin-top: 0;
+  margin-bottom: 17px;
```

Comment:
try to set only margin bottom inside elements. if some elements have default margins, just reset it